homeassistant/config.py

```diff
@@ -576,10 +582,25 @@ async def async_process_ha_core_config(hass: HomeAssistant, config: dict) -> Non
         (CONF_MEDIA_DIRS, "media_dirs"),
         (CONF_LEGACY_TEMPLATES, "legacy_templates"),
         (CONF_CURRENCY, "currency"),
+        (CONF_COUNTRY, "country"),
+        (CONF_LANGUAGE, "_language"),
     ):
         if key in config:
             setattr(hac, attr, config[key])
 
+    if hac.language is None:
+        owner = await hass.auth.async_get_owner()
+        if owner is not None:
+            # pylint: disable-next=import-outside-toplevel
+            from .components.frontend import storage as frontend_store
+
+            _, owner_data = await frontend_store.async_user_store(hass, owner.id)
```

Comment:
Is this correct, btw..? `async_user_store` returns `data`, not `data[owner.id]`?

Reply:
Thanks @akx, the code was indeed incorrect 👍